jsapp/js/account/usage/usage.component.tsx

```diff
@@ -45,7 +45,10 @@ export default function Usage() {
   const location = useLocation();
 
   const isFullyLoaded = useMemo(
-    () => usage.isLoaded && productsContext.isLoaded && limits.isLoaded,
+    () =>
```

Comment:
This dependency array needs to include `limits.stripeEnabled`, since it's referenced in the memo function. (I keep footgunning myself with this.) Once that's done, feel free to merge!